src/Components/ResourceList/Documentation/ResourceCard.stories.tsx

```diff
@@ -115,56 +115,53 @@ resourceListCardStories.add(
           contextMenuInteraction={select(
             'contextMenuInteraction',
             ['alwaysVisible', 'showOnHover'],
-            'showOnHover'
+            'alwaysVisible'
           )}
           style={object('style', resourceCardExampleStyle)}
-          direction={
-            FlexDirection[
-              select('direction', mapEnumKeys(FlexDirection), 'Column')
-            ]
-          }
-          alignItems={
-            AlignItems[select('alignItems', mapEnumKeys(AlignItems), 'Stretch')]
-          }
-          justifyContent={
-            JustifyContent[
-              select(
-                'justifyContent ',
-                mapEnumKeys(JustifyContent),
-                'FlexStart'
-              )
-            ]
-          }
-          margin={
-            ComponentSize[select('margin', mapEnumKeys(ComponentSize), 'Small')]
-          }
```

Comment:
Flexdirection is actually a pretty common prop being used for some cards. It seems like we dropped support for that and that is fine because we are consolidating the cards anyway but let's walk through this together